src/main/scala/is/hail/expr/AnnotationImpex.scala

```diff
@@ -419,10 +428,11 @@ object TableAnnotationImpex extends AnnotationImpex[Unit, String] {
         case t: TStruct => JsonMethods.compact(t.toJSON(a))
         case TInterval(TLocus(gr, _), _) =>
           val i = a.asInstanceOf[Interval]
-          if (i.start.asInstanceOf[Locus].contig == i.end.asInstanceOf[Locus].contig)
+          val bounds = if (i.start.asInstanceOf[Locus].contig == i.end.asInstanceOf[Locus].contig)
             s"${ i.start }-${ i.end.asInstanceOf[Locus].position }"
           else
             s"${ i.start }-${ i.end }"
```

Comment:
I think we should simplify this and just return `i.start-i.end` rather than `contig:start-end` if it's the same contig. @tpoterba do you have any concerns?

Reply:
doesn't that mean you no longer know what contig it's on, though?

Reply:
The contig will just get repeated twice:
So "1:10-1000" would be "1:10-1:1000".

Reply:
oh, you mean get rid of the special casing when they're on the same contig. Got it.